breaker-rails-cache-repo.gemspec

```diff
@@ -14,16 +14,16 @@ Gem::Specification.new do |spec|
   end
 
   spec.summary       = %q{This gem is a memcached implementation of the repo pattern used by the breaker gem}
-  spec.description   = %q{Dalli based repo for the breaker gem}
+  spec.description   = %q{Rails Cache backed repo for the breaker gem}
   spec.license       = "MIT"
 
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
-  spec.add_dependency "dalli"
   spec.add_dependency "breaker"
```

Comment:
Did you consider locking down breaker to the major version you wrote your extension against?